cli/src/lib.rs

```diff
@@ -3328,7 +3363,84 @@ fn validator_flags(
     Ok(flags)
 }
 
+// Returns Surfpool flags.
+// This flags will be passed to the Surfpool, it allows to configure the validator.
+fn surfpool_flags(
+    cfg: &WithPath<Config>,
+    surfpool_config: &Option<SurfpoolConfig>,
+    tui: bool,
```

Comment:
I think this should be renamed to reflect more - it's not necessarily just "tui", but if we're starting a full localnet or a "lighter" one for tests.

For example, if it's for tests we want to disable instruction profiling, set the max profiles to 0, and disable studio. However, if we're in full simnet mode (for `anchor localnet`), we want all features:
```rust
if !full_simnet_mode {
    flags.push("--no-tui");
    flags.push("--disable-instruction-profiling");
    flags.push("--max-profiles");
    flag.push("1");
    flags.push("--no-studio");
}
```